upload-all.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python
+"""
+Cross-platform script to upload Wikipedia Android APKs to releases server.
+
+Usage:
+    python upload-all.py [options]
+
+Options:
+    --verbose    Show detailed output during upload
+    --help       Show this help message
+
+Requirements:
+    - SCP must be available in PATH (or specify path with --scp-path)
+    - SSH key authentication must be set up for releases.discovery.wmnet
+    - APK files must exist in the releases/ directory
+
+The script uploads:
+    - Beta APKs (wikipedia-*-beta*.apk) to betas/ directory
+    - Stable APKs (wikipedia-*-r*.apk) to stable/ directory
+"""
+
+import argparse
+import glob
+import os
+import platform
+import subprocess
+import sys
+
+# Base configuration
+RELEASES_DIR = 'releases'
+REMOTE_HOST = 'releases.discovery.wmnet'
+REMOTE_BASE_PATH = '/srv/org/wikimedia/releases/mobile/android/wikipedia'
+BETA_REMOTE_PATH = f'{REMOTE_BASE_PATH}/betas/'
+STABLE_REMOTE_PATH = f'{REMOTE_BASE_PATH}/stable/'
+
+
+def find_scp_command():
+    """Find the SCP command on different platforms"""
+    if platform.system() == 'Windows':
+        # On Windows, try common locations for SCP
+        possible_paths = [
+            'scp',  # If it's in PATH
+            'C:/Windows/System32/OpenSSH/scp.exe',
+            'C:/Program Files/Git/usr/bin/scp.exe',
+            'C:/Program Files (x86)/Git/usr/bin/scp.exe'
```

Comment:
A bit of concern on these specific `.exe` paths, maybe we can remove them from the script.